src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
```

Comment:
You don't need to wrap everything in an additional `div`, the `container` class can be specified for the `body`.